lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerPage.pm

```diff
@@ -119,4 +125,10 @@ sub select_volume_group {
     return $self->select_item_in_system_view_table('LVM Volume Groups|' . $vg);
 }
 
+sub select_device_in_table {
```

Comment:
As per our discussion, could you please change this method so we could revert workaround once we get selection activating menu? selection of the row sub-item like `/dev/sda|sda` should already work.